src/simple_logger.rs

```diff
@@ -1,14 +1,41 @@
 #[cfg(not(test))]
-use crate::ios::ios_interface::{CoreLogLevel, CoreLogMessageThreadSafe, LOG_SENDER};
 #[cfg(test)]
 use chrono::Local;
 #[cfg(not(test))]
 use chrono::Utc;
 use log::*;
-use std::sync::Once;
+use std::fmt;
+use std::sync::{mpsc::Sender, Once};
 
 static INIT: Once = Once::new();
 
+pub static mut IOS_LOG_SENDER: Option<Sender<CoreLogMessageThreadSafe>> = None;
```

Comment:
Moved the non-iOS specific log things here. Not sure about `SENDER`. This is still coupled with iOS. For now it's just to make it compile.

Reply:
SENDER should be used for both platforms (unless if threte is avery good reason not to do so).

Reply:
Fixed in https://github.com/Co-Epi/app-backend-rust/pull/83